src/js/components/PlacementConstraintsPartial.js

```diff
@@ -152,27 +129,27 @@ export default class PlacementSection extends Component {
               placeholder="hostname"
               value={constraint.fieldName}
             />
+            <FieldHelp>
+              E.g hostname.
+            </FieldHelp>
             <FieldError>{fieldNameError}</FieldError>
           </FormGroup>
           <FormGroup
-            className={{
-              "column-4": !hideValueColumn,
-              hidden: hideValueColumn
-            }}
+            className={commonFieldsClassNames}
             required={valueIsRequired}
             showError={Boolean(valueError)}
           >
             {valueLabel}
-            <FieldInput
-              className={{ hidden: valueIsRequiredEmpty }}
-              name={`constraints.${index}.value`}
-              type="text"
-              value={constraint.value}
-            />
-            <FieldHelp
-              className={{ hidden: valueIsRequired || valueIsRequiredEmpty }}
-            >
-              This field is optional
+            {valueIsRequiredEmpty
+              ? this.getToolTip(
+                  fieldValue,
+                  OperatorTypes[constraint.operator].tooltipContent
```

Comment:
As I said above, maybe something like
`The ${constraint.operator} operator does not accept a value.`?